### PR TITLE
Added Arya Soni

### PR DESCRIPTION
**Pull request by aryasoni98**

### CODE REVIEW

**Comment by praveenscience** [comment]
Still conflicts are there, would you like to rebase and send?

File: data.json [hunk 1]
```diff
@@ -665,10 +665,18 @@
...
+    "issueNumber": 354
+  },
   {
     "name": "Cas Martens",
     "githubUsername": "DevCassie",
     "imageUrl": "https://user-images.githubusercontent.com/19227530/110542942-fa6cab00-8129-11eb-9835-d482375dae8a.png",
     "issueNumber": 394
+
   }
+
```
Comment (praveenscience) [inline data.json:681]: Remove the empty lines if you can.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (aryasoni98) [reply data.json]: Done ✅

**Review by praveenscience — APPROVED** [review]
LGTM 👍

**Pull request merged**